examples/track.rs

```diff
@@ -57,13 +86,20 @@ fn main() {
 
     dispatcher.setup(&mut world.res);
 
-    for _ in 0..10000 {
+    for _ in 0..50 {
         world.create_entity().with(TrackedComponent(0)).build();
     }
 
     dispatcher.dispatch(&mut world.res);
     world.maintain();
 
+    let entities = (&world.entities(), &world.read_storage::<TrackedComponent>()).join().map(|(e, _)| e).collect::<Vec<Entity>>();
```

Comment:
This line is kinda gnarly, but I cannot think how to make it better.

Reply:
.map(|t| t.0).collect::<Vec<_>>()

Reply:
or you can keep the "e" and just change the collect

Reply:
eh, I think it is clearer that it is de-structuring the tuple of the join and it only wants the entities. t.0 isn't very clear. If anything I'll just move the entities/read_storage to a line above it.